src/autorest-core/lib/pipeline/plugins/tree-shaker.ts

```diff
@@ -1,7 +1,8 @@
-import { AnyObject, DataHandle, DataSink, DataSource, Node, parseJsonPointer, Transformer, QuickDataSource } from '@microsoft.azure/datastore';
+import { AnyObject, DataHandle, DataSink, DataSource, Node, parseJsonPointer, Transformer, QuickDataSource, JsonPath } from '@microsoft.azure/datastore';
 import { ConfigurationView } from '../../configuration';
 import { PipelinePlugin } from '../common';
 import { clone } from '@microsoft.azure/linq';
+import { retries } from 'mocha-typescript';
```

Comment:
? What's `retries` in there for? that won't work at runtime...

Reply:
it probably got imported on the fly. Didn't realize that. I
 am gonna remove it.